testing/readme.md

```diff
@@ -0,0 +1,58 @@
+# OPAE Testing Framework #
+
+## Objective ##
+
+The main goal of the testing framework used by OPAE is to enable (Google test)
+tests that don't require the presence of either an OPAE kernel driver or an
+OPAE compatible FPGA. An additional goal is to allow authoring tests that can
+easily ported from a mock environment to a real system.
```

Comment:
Is porting strictly required to run these on hardware, or is it a matter of re-linking?

Reply:
I think you could run most (if not all) of these tests on real HW by omitting the mock.c file and making sure the paths aren't re-rooted.